### PR TITLE
Prepare release 1.5.0

### PR DESCRIPTION
**Pull request by lmolkova**
- profiles: Remove unused `Label` definition. [#602](https://github.com/open-telemetry/opentelemetry-proto/pull/602)
- all: Add note about `schema_url` format (including version). [#605](https://github.com/open-telemetry/opentelemetry-proto/pull/605)
- logs: Add top-level `event_name` field to log records instead of `event.name` attribute. [#600](https://github.com/open-telemetry/opentelemetry-proto/pull/600)
- profiles: drop duplicate `attributes` field in message Profile. [#606](https://github.com/open-telemetry/opentelemetry-proto/pull/606)

### CODE REVIEW

**Review by Kielek — APPROVED** [review]
🎉

**Review by cijothomas — APPROVED** [review]

**Review by trask — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,16 @@ The full list of changes can be found in the compare view for the respective rel
 
 ### Added
 
+### Changed
+
+## 1.5.0 - 2024-12-12
```
Comment (trask) [inline CHANGELOG.md:11]:
```suggestion
## 1.5.0 - 2024-12-13
```

**Review by jsuereth — APPROVED** [review]

**Review by reyang — APPROVED** [review]

**Review by pellared — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,7 +6,16 @@ The full list of changes can be found in the compare view for the respective rel
 
 ### Added
 
+### Changed
+
+## 1.5.0 - 2024-12-13
+
+### Added
+
+- profiles: Remove unused `Label` definition. [#602](https://github.com/open-telemetry/opentelemetry-proto/pull/602)
```
Comment (pellared) [inline CHANGELOG.md:15]: Should it not be under ```### Removed`?

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,7 +6,16 @@ The full list of changes can be found in the compare view for the respective rel
...
+- all: Add note about `schema_url` format (including version). [#605](https://github.com/open-telemetry/opentelemetry-proto/pull/605)
 - logs: Add top-level `event_name` field to log records instead of `event.name` attribute. [#600](https://github.com/open-telemetry/opentelemetry-proto/pull/600)
+- profiles: drop duplicate `attributes` field in message Profile. [#606](https://github.com/open-telemetry/opentelemetry-proto/pull/606)
```
Comment (pellared) [inline CHANGELOG.md:18]: Should it not be under ```### Removed`?

**Review by pellared — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (lmolkova) [reply CHANGELOG.md]: sure, pedant 😅

**Comment by chalin** [comment]
@lmolkova - do the release instructions for this repo also mention https://github.com/open-telemetry/opentelemetry.io/actions/workflows/build-dev.yml? Cf. https://github.com/open-telemetry/semantic-conventions/issues/1009#issuecomment-2541965117

I just ran it now and all seems fine (two external links need to be added to the refcache, but that's not a build error): https://github.com/open-telemetry/opentelemetry.io/actions/runs/12325743342/job/34405496807

**Review by pellared — APPROVED** [review]

**Review by marcalff — APPROVED** [review]

**Pull request merged**